### PR TITLE
16 use the auto flattening of the data to make a connection and add new form

### PR DESCRIPTION
**Pull request by Abhishek-N**
Fix: #16

### CODE REVIEW

File: macros/flatten_json.sql
Comment (siddhant3030) [inline macros/flatten_json.sql:1]: any reason for changing the folder names? you could've just change the target in profiles.yml file to dev?

File: models/dev_staging/dev_staging_ethiopia.sql [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{{ config(
+  materialized='table'
+) }}
+
+{{ flatten_json(
```
Comment (siddhant3030) [inline models/dev_staging/dev_staging_ethiopia.sql:5]: we can merge both the models into one using cte. we dont need this extra step

File: models/dev_intermediate/classroom_surveys/schema.yml
Comment (siddhant3030) [inline models/dev_intermediate/classroom_surveys/schema.yml:1]: we can add more tests here. identify those like date and form fields

File: models/dev_staging/dev_staging_ethiopia.sql (hunk 1 above)
Comment (Abhishek-N) [reply models/dev_staging/dev_staging_ethiopia.sql:5]: tried doing so but we can't add cte when we use dbt_utils.star. so i took this approach

**Comment by Abhishek-N** [comment]
@siddhant3030 please check if this looks correct

**Pull request merged**